src/parser/TraverseSentences.cpp

```diff
@@ -222,4 +222,30 @@ std::string YieldSentence::toString() const {
     }
     return buf;
 }
+
+std::string GetSubgraphSentence::toString() const {
+    std::string buf;
+    buf.reserve(256);
+    buf += "GET SUBGRAPH ";
+    if (step_ != nullptr) {
+        buf += step_->toString();
+    }
+    if (from_ != nullptr) {
+        buf += " ";
+        buf += from_->toString();
+    }
+    if (in_ != nullptr) {
```

Comment:
There is the `in` , `out` and `both` information already.

Reply:
pls be more specific